e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/GlobalSearchAO.java

```diff
@@ -194,5 +215,16 @@ public boolean apply(final WebElement element) {
                 }
             };
         }
+
+        public SearchResultItemPreviewAO checkTags(String ... list) {
+            String tags = String.format("%s %s %s %s", get(TAGS).find(By.xpath("./span[2]/span")).text(),
+                    get(TAGS).find(By.xpath("./span[3]")).text(),
+                    get(TAGS).find(By.xpath("./span[4]")).text(),
+                    get(TAGS).find(By.xpath("./span[5]")).text());
+            for( String i:list) {
+                tags.contains(i);
+            }
```

Comment:
Can be replaced by Arrays.stream(list).forEach(tags::contains)

Reply:
Done